packages/expo-modules-core/ios/Tests/ExceptionsSpec.swift

```diff
@@ -73,7 +73,7 @@ final class ExceptionsSpec: ExpoSpec {
       let b = TestException().causedBy(a)
       let c = TestException().causedBy(b)
 
-      expect(c.rootCause) === a
+      expect((c.rootCause as! TestException) === a).to(beTrue())
```

Comment:
```suggestion
      expect((c.rootCause as! TestException)).to(beEqual(a))
```
Something like this possible? (Suggestion is not real code)